manifests/vrrp/instance.pp

```diff
@@ -150,6 +153,7 @@
   $auth_type                  = undef,
   $auth_pass                  = undef,
   $track_script               = undef,
+  Optional[Array[String]] $track_process = undef,
```

Comment:
Please default to an empty array and enforce the minimal string length:
```suggestion
  Array[String[1]] $track_process = undef,
```

please also ensure that all the `=` are aligned.

Reply:
I can enforce minimum String length, but this must remain Optional and default to undef, or the keepalived.conf file will have empty track_process { } blocks in it.

Reply:
you can use this in the themplate
```
<% unless @track_process? then -%>
```

Reply:
Why run the template at all if we can shortcut out early using Optional/undef?  I don't understand the advantage of running more code and "unless"ing out?